src/apps/Map/components/SearchBar.js

```diff
@@ -7,6 +7,7 @@ import MenuButton from '~/components/MenuButton';
 import config from '~/config';
 import Store from '~/store';
 import { media } from '~/styles/utils';
+import SearchIconImage from '~/images/search.png';
```

Comment:
That's correct :D
I've tried this. But I didn't get it working  - then I've dumped the idea. Styled components always inserted a JavaScript object into the `backgound-image` url. So I would be glad, if you know how to do it ;)

Reply:
We are using [`react-svg-loader`](https://www.npmjs.com/package/react-svg-loader) in our `webpack.common.js` file. All SVG files are loaded as JSX component. Therefore, as I can see, we cannot use SVG as `background-image`.

I will try to redo this as SVG Component instead of `background-image`.

Reply:
Changed it.